docs/elements/hx-icon/index.html

```diff
@@ -1,5 +1,7 @@
 ---
-title: <hx-icon>
+title: Icons
+also:
+  styleguide/iconography: Icons
```

Comment:
Also, is there a component page for icons? Or just style guide and element pages?

Reply:
after speaking with Drew, we decided that icons be moved to styleguide instead of components and be renamed to iconography

Reply:
after speaking with Drew, moved icons to styleguide under iconography folder :)